src/index.ts

```diff
@@ -19,6 +19,9 @@ process
 dotenv.config();
 const app = express();
 
+// Start the OpenTelemetry SDK to collect traces in Langfuse
+telemetrySDK.start();
```

Comment:
Let's make sure that the application can start also without the observability stack